.github/pull_request_template.md

```diff
@@ -0,0 +1,42 @@
+
+<!-- Thanks for submitting this pull request for KeY -->
+<!-- Since the project has a strict review policy, please make the -->
+<!-- reviewer's job easier by providing the necessary information -->
+<!-- in the text below. The comments may remain since they will be -->
+<!-- invisible when showing the PR. -->
+
+## Related Issue
+
+<!-- Please remove if this PR is not related to an issue. -->
+<!-- Please add number if it is in answer to an issue. -->
+This pull request answers to issue #.
+
+## Intended Change
+<!-- Please give a brief description of what behaviour changes and 
+     why it should be changed. -->
+
+## Reason for the pull request
+
+<!-- one item like: 
+     bugfix, new feature, refactoring, repaired workflow, ... -->
+
+## Touched elements
+
+<!-- A brief overview of the touched code areas. No need to list 
+     files (that is available in github). Mention things like:
+     TacletApps in key.core, ANTLR for JML, github config, ... -->
+     
+## Information for the reviewer
+    
+<!-- Additional information to help the reviewer understand the request -->
+
+## Additional contact(s)
+
+<!-- who apart from yourself is involved in this pull request?
+     Use @mentions to refer to them here -->
+     
+<!-- DRAFT MODE: Please note that on the button to submit this pull
+     request you can select between submitting a merge-ready request
+     or one in draft mode (still evolving). 
+     Please use the draft mode unless you think that your proposal
+     should be brought onto master in the current form. -->
```

Comment:
Maybe we want additional information, e.g., to create a good changelog.

https://github.com/stevemao/github-issue-templates/blob/master/checklist2/PULL_REQUEST_TEMPLATE.md

Reply:
Also required may be the consideration: "The documentation is/will be updated".

Reply:
I do not really believe in Changelogs. Is that not usually a very obvious reason for merge conflicts?

What does "The documentation is/will be updated" mean? I do not understand this point.

Reply:
> What does "The documentation is/will be updated" mean? I do not understand this point.
The author adds a note, whether this Pull Request results are also reflected in `key-docs`. 

I think changelog are important for a real project (just look at dafny, z3, cbmc, ...).